tests/cnt-i18n_xhtml-xml-lang/EPUB/package.opf

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This test only addresses XHTML content, but the spec says that xml:lang must be tested in the package etc too. Should that also be in this test? (nav.xhtml too?) Or is that already covered in one of your existing tests, which just needs to be tagged to https://w3c.github.io/epub-specs/epub33/rs/#confreq-rs-xml-lang?